app/views/invoice_mailer/notify_user_email.en.html.erb

```diff
@@ -0,0 +1,15 @@
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
```

Comment:
I believe you don't need this, since we have the [mailer layout](https://github.com/comarev/comarev/blob/main/app/views/layouts/mailer.html.erb)